tabs/firmware_flasher.html

```diff
@@ -55,7 +55,8 @@
                         <td><span class="description" i18n="firmwareFlasherManualBaudDescription"></span></td>
                     </tr>
                     <tr class="option noboarder">
-                        <td><label> <input class="show_development_releases toggle" type="checkbox" /> <span
+                        <!-- woga65: default to show development releases -->
+                        <td><label> <input class="show_development_releases toggle" type="checkbox" checked/> <span
```

Comment:
I'm assuming that this is for your testing only. Development releasees should **not** be shown by default.

Reply:
Yes, for testing. I will clean this up as soon as all probebly needed changes are discussed and applied.